addons/rating/static/src/messaging_menu_patch.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <templates xml:space="preserve">
     <t t-inherit="mail.MessagingMenu.content" t-inherit-mode="extension">
-        <xpath expr="//NotificationItem[@body='message?.inlineBody']" position="attributes">
+        <xpath expr="//NotificationItem[@body='message?.inlineBody or message?.subtypeDescription']" position="attributes">
```

Comment:
This is not great that xpath relies on code, we should find something better. For example a more specific parent, like the t-foreach, and adding a name= to it.

Reply:
Updated, I didn't need to add a name attribute